main.go

```diff
@@ -163,40 +159,36 @@ func main() {
 		requiredTypes[strings.ToLower(t)] = true
 	}
 
-	for _, p := range ppl {
-		if !p.Active {
-			continue
-		}
-		if !p.Invited {
-			continue
-		}
-		if !p.InAuditScope {
+	for _, info := range ppl {
+		if !requiredTypes[info["employee_type"]] {
 			continue
 		}
 
-		eType := strings.ToLower(p.EmployeeType)
-		if !requiredTypes[eType] {
-			continue
-		}
+		// Instructions to be populated based on the user's onboarding status
+		var needs []string
 
-		needs := []string{}
-		if !p.PoliciesAccepted {
+		// Account for users with training that has not started or with outstanding security training
+		switch info["onboarding_status"] {
```

Comment:
I'm not sure what the possible values for this are. When testing, I only saw `all_tasks_completed`, `not_started`, and `security_training`.